src/components/Completed.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import { CircularProgressbar } from 'react-circular-progressbar';
+
+function Completed() {
+  return (
+    <>
+      <div className="main-completed">
```

Comment:
- [ ] Please bring progress container closer to the chapter container as seen in the original design.
### Your Design
![image](https://github.com/manzitresor/book_store/assets/45460192/5cadab79-bc9e-4dce-8443-0139e07a9307)
### Original design
![image](https://github.com/manzitresor/book_store/assets/45460192/8e488385-a5a1-4919-aa7f-6bc4954ca924)